lib/velum/suse_connect.rb

```diff
@@ -78,6 +79,17 @@ def regcode
                         smt_insecure: smt_insecure,
                         credentials:  credentials).regcode
       end
+
+      def human_error_for_exception(exception)
```

Comment:
instead of abstracting the messages outside of the error class itself into a helper method, why not add the messages to the class itself, that we define above?

```ruby
class MissingRegCodeException < StandardError
  def initialize(msg="Missing registration code")
    super
  end
end
```

Reply:
Updated but in a bit different style, as we don't really expect `msg` to be settable in this case.